_index.md

```diff
@@ -18,130 +18,87 @@ summary: TiDB is an open-source distributed SQL database that supports Hybrid Tr
-
-[Tune SQL Performance](https://docs.pingcap.com/tidb/dev/sql-tuning-overview)
-
+<p><ExternalLink to="./performance-tuning-overview">Tuning Overview</ExternalLink></p>
+<p><ExternalLink to="./performance-tuning-methods">Tuning Methods</ExternalLink></p>
+<p><ExternalLink to="./performance-tuning-practices">Tune OLTP Performance</ExternalLink></p>
+<p><ExternalLink to="./tune-operating-system">Tune Operating System</ExternalLink></p>
+<p><ExternalLink to="./configure-memory-usage">Tune Configurations</ExternalLink></p>
+<p><ExternalLink to="./sql-tuning-overview">Tune SQL Performance</ExternalLink></p>
 </LearningPath>
 
 <LearningPath label="Tools" icon="doc7">
 
-[TiUP](https://docs.pingcap.com/tidb/dev/tiup-overview)
-
-[TiDB Operator](https://docs.pingcap.com/tidb/dev/tidb-operator-overview)
-
-[TiDB Data Migration (DM)](https://docs.pingcap.com/tidb/dev/dm-overview)
-
-[TiDB Lightning](https://docs.pingcap.com/tidb/dev/tidb-lightning-overview)
-
-[Dumpling](https://docs.pingcap.com/tidb/dev/dumpling-overview)
-
-[TiCDC](https://docs.pingcap.com/tidb/dev/ticdc-overview)
-
-[Backup & Restore (BR)](https://docs.pingcap.com/tidb/dev/backup-and-restore-overview)
-
-[PingCAP Clinic](https://docs.pingcap.com/tidb/dev/clinic-introduction)
-
+<p><ExternalLink to="./tiup-overview">TiUP</ExternalLink></p>
+<p><ExternalLink to="./tidb-operator-overview">TiDB Operator</ExternalLink></p>
+<p><ExternalLink to="./dm-overview">TiDB Data Migration </ExternalLink></p>
+<p><ExternalLink to="./tidb-lightning-overview">TiDB Lightning</ExternalLink></p>
+<p><ExternalLink to="./dumpling-overview">Dumpling</ExternalLink></p>
+<p><ExternalLink to="./ticdc-overview">TiCDC</ExternalLink></p>
+<p><ExternalLink to="./backup-and-restore-overview">Backup & Restore </ExternalLink></p>
+<p><ExternalLink to="./clinic-introduction">PingCAP Clinic</ExternalLink></p>
 </LearningPath>
 
 <LearningPath label="Reference" icon="cloud-dev">
 
-[TiDB Roadmap](https://docs.pingcap.com/tidb/dev/tidb-roadmap)
-
-[TiDB Configuration File Parameters](https://docs.pingcap.com/tidb/dev/tidb-configuration-file)
-
-[TiDB Command-line Flags](https://docs.pingcap.com/tidb/dev/command-line-flags-for-tidb-configuration)
-
-[TiDB Control](https://docs.pingcap.com/tidb/dev/tidb-control)
-
-[System Variables](https://docs.pingcap.com/tidb/dev/system-variables)
-
-[Release Notes](https://docs.pingcap.com/tidb/dev/release-notes)
-
-[FAQ Summary](https://docs.pingcap.com/tidb/dev/faq-overview)
-
+<p><ExternalLink to="./tidb-roadmap">TiDB Roadmap</ExternalLink></p>
+<p><ExternalLink to="./tidb-configuration-file">TiDB Configuration File Parameters</ExternalLink></p>
+<p><ExternalLink to="./command-line-flags-for-tidb-configuration">TiDB Command-line Flags</ExternalLink></p>
+<p><ExternalLink to="./tidb-control">TiDB Control</ExternalLink></p>
+<p><ExternalLink to="./system-variables">System Variables</ExternalLink></p>
+<p><ExternalLink to="./release-notes">Release Notes</ExternalLink></p>
```

Comment:
```suggestion
<p><ExternalLink to="../../tidb/dev/release-notes">Release Notes</ExternalLink></p>
```

Reply:
reason: The Roadmap doc is only available in dev version: https://docs.pingcap.com/tidb/dev/tidb-roadmap. After a new branch (for example, v9.0) is created based on the master, we can avoid updating the link here for Roadmap.